core-data/src/androidMain/kotlin/io/github/droidkaigi/confsched2022/data/DriverFactory.kt

```diff
@@ -0,0 +1,11 @@
+package io.github.droidkaigi.confsched2022.data
+
+import android.content.Context
+import com.squareup.sqldelight.android.AndroidSqliteDriver
+import com.squareup.sqldelight.db.SqlDriver
+
+actual class DriverFactory(private val context: Context) {
```

Comment:
I'd say we need to use an interface instead of "expect actual" .
https://github.com/DroidKaigi/conference-app-2022/pull/286#issuecomment-1240914531
e: /Users/runner/work/conference-app-2022/conference-app-2022/core-data/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/data/DriverFactory.kt: (5, 14): Expected class 'DriverFactory' has no actual declaration in module <core-data_debug> for JVM
> Task :core-data:compileDebugUnitTestKotlinAndroid FAILED